docusaurus.config.js

```diff
@@ -435,19 +435,19 @@ const config = {
                 label: 'Flowscan Testnet',
               },
               {
-                to: '/node-ops/node-operation/past-sporks/',
+                to: '/references/run-and-secure/node-operation/past-sporks/',
```

Comment:
don't we need to add new redirects and not change the existing ones?

Reply:
This is just the footer configuration.  Imo it makes sense to update to new location instead of relying on redirect, but we could have.

However, you're right, I did miss a redirect :P

Reply:
You're right I was thinking it's the redirect file